.github/workflows/main.yaml

```diff
@@ -200,3 +200,61 @@ jobs:
       - run: make edgesite-server-image ARCH="amd64"
 
       - run: make iptablesmgrimage
+
+  helm-ci:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 30
+    name: Test helm install and uninstall
+    env: 
+      CLUSTER_NAME: kubeedge-chart
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        with:
+          config: .github/workflows/kind/kind.yaml
+          cluster_name: ${{ env.CLUSTER_NAME }}
+      
+      - name: Set environmental variables
+        run: |
+          NODE_IP=$(kubectl get nodes -o jsonpath={.items[0].status.addresses[0].address})
+          echo "NODE_IP=$NODE_IP" >> $GITHUB_ENV
+          echo "IMAGE_TAG=$(date +%Y%m%d)" >> $GITHUB_ENV
+
+      - name: Build cloudimage
+        run: |
+          make cloudimage -e IMAGE_TAG=${IMAGE_TAG}
+          kind load docker-image kubeedge/cloudcore:${IMAGE_TAG} --name ${CLUSTER_NAME}
+      
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Setting up kubeedge cloudcore
+        run: |
+          cd build/helm && helm upgrade --install cloudcore ./cloudcore --namespace kubeedge --create-namespace -f ./cloudcore/values.yaml --set cloudCore.modules.cloudHub.advertiseAddress[0]=${NODE_IP} --set cloudCore.image.tag=${IMAGE_TAG}
+      
+      - name: Waiting for 10 seconds to keep cloudcore installed
+        uses: jakejarvis/wait-action@master
+        with:
+          time: '10s'
+
+      - name: Check cloudcore status
+        run: |
+          status=`kubectl get pods -n kubeedge | grep -i "cloudcore" |awk '{ print $3 }'`
+          if [ "$status" != "Running" ]; then
```

Comment:
whether we need to retry, just in case that in 10s, cloudcore still doesn't start.

Reply:
Maybe we need `--wait --timeout 30`,  this step will be removed. And the image have pushed to the kind cluster, there will be no blocks to set up.

Reply:
I wonder if we can write a script to do this and only call the script in github workflow?

Reply:
Reuse codes in keadm-e2e to implement the status checking and edge joining ？

Reply:
Yes, it will me more easy to maintain if in future we switch to other ci platform and can reuse the script.

Reply:
OK. I will have a test and make a re-pull request here. Currently, it got some bug.

Reply:
OK sounds good :)